index.bs

```diff
@@ -578,6 +564,11 @@ but if the provided [=sensor readings=] are product of [=sensor fusion=] perform
 in software, the [=platform sensor=] corresponds to a set of [=device sensors=]
 involved in the [=sensor fusion=] process.
 
+Discrepancies between a [=sensor reading=]
+and the corresponding physical quantities being measured
```

Comment:
physical quantity?

Reply:
could it be several "quantities" for the same sensor? (e.g. acceleration and angular velocity for orientation sensor)

Reply:
Leaving this as is.